src/renderer/entities/wallet/ui/DerivedAccount/DerivedAccount.tsx

```diff
@@ -0,0 +1,93 @@
+import { BodyText, FootnoteText, Icon, InfoPopover } from '@renderer/shared/ui';
+import { cnTw } from '@renderer/shared/lib/utils';
+import { IconNames } from '@renderer/shared/ui/Icon/data';
+import { KeyType } from '@renderer/shared/core';
+import { useI18n } from '@renderer/app/providers';
+import { useDerivedInfo } from '../../lib/useDerivedInfo';
+
+const KeyIcon: Record<KeyType, IconNames> = {
+  [KeyType.CUSTOM]: 'ddCustom',
+  [KeyType.GOVERNANCE]: 'ddGovernance',
+  [KeyType.HOT]: 'ddHot',
+  [KeyType.MAIN]: 'ddMain',
+  [KeyType.PUBLIC]: 'ddPublic',
+  [KeyType.STAKING]: 'ddStaking',
+};
+
+const KeyTitle: Record<KeyType, string> = {
+  [KeyType.CUSTOM]: 'derivedAccount.customTitle',
+  [KeyType.GOVERNANCE]: 'derivedAccount.governanceTitle',
+  [KeyType.HOT]: 'derivedAccount.hotTitle',
+  [KeyType.MAIN]: 'derivedAccount.mainTitle',
+  [KeyType.PUBLIC]: 'derivedAccount.publicTitle',
+  [KeyType.STAKING]: 'derivedAccount.stakingTitle',
+};
+
+type ShardedKeyType = KeyType.CUSTOM | KeyType.GOVERNANCE | KeyType.MAIN | KeyType.STAKING;
```

Comment:
Maybe move it to account.ts near KeyType? I think we can use it in multiple places